src/issue-filing/github/github-url-rectifier.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export type UrlRectifier = (url: string) => string;
+
+export const rectify: UrlRectifier = (url: string): string => {
+    const ownerAndRepo = new RegExp('^https?://github.com/[^/]+/[^/]+$');
```

Comment:
Does this support ````https://www.github.com/owner/project```` ?

Reply:
Yes, there is a test case with `'http://github.com/me/my-repo'` (I'm guessing you're question points to the fact you have no ending `/`). You can see the test cases for more scenarios.

Reply:
I'm actually thinking the case where user put 'www' before 'github.com', should we support that?

Reply:
In that case, the user is responsible to provide the correct url. Also remember we have a placeholder with and example url which doesn't have `www` in it. That being said, if the user is on github enterprise account where the url start with `www` this rectifier will not change the url but it won't "throw" any error neither.